content/operate/redisinsight/install/install-on-desktop.md

```diff
@@ -8,14 +8,24 @@ linkTitle: Install on desktop
 title: Install on desktop
 weight: 1
 ---
-Install desktop Redis Insight to use it in development.
-
 ## Supported operating systems
 
 Redis Insight is supported on multiple operating systems:
-- Windows
-- Linux
-- macOS
+
+| Operating System | Supported Versions |
+|:---              |:---                |
+| **Windows**      | Windows 11         |
+|                  | Windows 10         |
+| **macOS**        | macOS 14           |
```

Comment:
just checked - macOS 15 should also be supported

maybe we can say something like:
Windows 10 and newer,
macOS 11 and newer, etc.?